mlir/test/CMakeLists.txt

```diff
@@ -247,4 +247,5 @@ add_lit_testsuite(check-mlir "Running the MLIR regression tests"
 
 add_lit_testsuites(MLIR ${CMAKE_CURRENT_SOURCE_DIR}
   DEPENDS ${MLIR_TEST_DEPENDS}
+  SKIP python
```

Comment:
I'm not quite following the effect of not doing it actually. 
What happens if I take this patch and remove just this line?

Reply:
> nothing? 

I mean surely this line is added for having an effect, otherwise we can just remove it right?

>  so no subdirs will be skipped.

I get that, I just don't quite figure what is the effect of skipping it. Can you describe a command I can run and what I should observe before and after?

Reply:
Yes